docs/references/phasing.tex

```diff
@@ -1,111 +1,179 @@
 
-\subsection{Limitations of the MWA reference code}
-By comparing the values calculated by the MWA code at various stages to values calculated using various \texttt{astropy} conversions, the authors of this memo determined that the MWA code is only correcting the u, v, w coordinates for precession and nutation and not for aberration. This is supported by a few lines of evidence, one is that the uvw correction is done using a rotation matrix, which cannot account for length contraction and the other is that the corrected values closely match the values generated using \texttt{astropy} conversions to the GCRS frame, but not to conversions to the ICRS frame (which differ by the inclusion of the effects of earth's motion around the sun, also called aberration).
+Taken all together, the projected baseline vectors, $\mathbf{r}^{\prime}$, can be represented as:
+\begin{equation}
+\mathbf{r}^{\prime} = \mathcal{R}_{\phi_{\rm FPA},x} \mathcal{R}_{\delta_{\rm PC}, y} \mathcal{R}_{\textrm{GHA}, z} \mathbf{r}_{\rm ant}
+\end{equation}
+\subsection{Projection}\label{ssec:projection}
+Once the $uvw$-coordinates are calculated, delay compensation for the $w$-projected distance is relatively straightforward, where the following per-visibility phase correction, $\phi^{\prime}$ is applied:
+\begin{equation}\label{eqn:projection}
+\phi^{\prime} = e^{-2i\pi\Delta w / \lambda_{\rm obs}},
+\end{equation}
+where $\Delta$w is the difference in $w$-projected distance (between the new and old phase center) for a given visibility, and $\lambda_{\rm obs}$ is the wavelength of the frequency channel the visibility belongs to. Note that for unprojected data, no phase term has been applied, so $\Delta w$ is the full distance from the baseline pair to the plane. Note that delay corrections are applied using the observed frequency, as stored in \texttt{UVData.freq\_array}, versus the rest-frame frequency of the object in question, which is not accounted for by pyuvdata.
```

Comment:
"Note that for unprojected data, no phase term has been applied, so $\Delta w$ is the full distance from the baseline pair to the plane." I found this a little confusing, maybe be more explicit, something like: "Note that when starting with unprojected data, no phase term has been applied, so $\Delta w$ is the full distance from the baseline pair to the plane, while if the data has been previously projected $\Delta w$ is the distance from the previous phase plane to the new phase plane."

Reply:
Yeah, that reads better -- swapped.